framework/auth/__init__.py

```diff
@@ -60,6 +63,13 @@ def register_unconfirmed(username, password, fullname):
             password=password,
             fullname=fullname)
         user.save()
+        mails.queue_mail(
+            to_addr=user.username,
+            mail=mails.NO_ADDON,
+            send_at=datetime.utcnow() + settings.NO_ADDON_WAIT_TIME,
+            user=user,
+            fullname=user.fullname
```

Comment:
Is `user` available in the mako context of the email template? If so, then `fullname` is unnecessary.